Dockerfile

```diff
@@ -17,13 +20,16 @@ RUN crudini --merge /etc/ironic/ironic.conf < /tmp/ironic.conf && \
 
 COPY ./runironic-api.sh /bin/runironic-api
 COPY ./runironic-conductor.sh /bin/runironic-conductor
+COPY ./runironic-exporter.sh /bin/runironic-exporter
 COPY ./rundnsmasq.sh /bin/rundnsmasq
 COPY ./runhttpd.sh /bin/runhttpd
 COPY ./runmariadb.sh /bin/runmariadb
 COPY ./configure-ironic.sh /bin/configure-ironic.sh
 
-# TODO(dtantsur): remove these 2 scripts if we decide to
-# stop supporting running all 2 processes via one entry point.
+# TODO(dtantsur): remove these 3 scripts and squash runexporterapp into
+# runironic-exporter if we decide to stop supporting running all 3 processes
+# via one entry point.
+COPY ./runexporterapp.sh /bin/runexporterapp
```

Comment:
Let's not re-add this, it was needed for backward compatibility.

Reply:
actually, shouldn't we leave this ?